### PR TITLE
main/zfs: upgrade to 0.8.2

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: main/zfs-vanilla/APKBUILD [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 # when changing _ver we *must* bump _rel
 _name=zfs
-_ver=0.8.1
-_rel=0
+_ver=0.8.2
+_rel=1
```
Comment (maxice8) [inline main/zfs-vanilla/APKBUILD]: shouldn't it be 0 ?

Comment (Cogitri) [reply main/zfs-vanilla/APKBUILD]: How would users be upgraded to the new version then?

**Comment by HRio** [comment]
0.8.2 fixes lots of bugs:
https://github.com/zfsonlinux/zfs/releases/tag/zfs-0.8.2
would be nice to see this in v3.10 as well...

**Comment by Cogitri** [comment]
Sure, once we have had this in edge for a few days, okay?

**Comment by Cogitri** [comment]
@ncopa mind taking a look?

**Comment by Cogitri** [comment]
Can some more people test this? I have spl's `z_rd_int` and `z_wr_iss` take up lots of CPU on my machine since the update.

**Comment by ncopa** [comment]
fixed with e92eab6d0422ed7960bb7800f04f76952c43728a

**Comment by HRio** [comment]
> Sure, once we have had this in edge for a few days, okay?
@Cogitri; I have loaded pinned @edge packages on a v3.10 system, to test on my side as well

**Comment by Cogitri** [comment]
Ah, thank you. Actually kind of forgot about that. Thanks for reminding me!

**Pull request closed without merging**